linera-service/src/linera/main.rs

```diff
@@ -1320,6 +1320,10 @@ struct ClientOptions {
     #[arg(long, env = "LINERA_CLIENT_TOKIO_THREADS")]
     tokio_threads: Option<usize>,
 
+    /// The number of Tokio blocking threads to use.
+    #[arg(long)]
+    tokio_blocking_threads: Option<usize>,
```

Comment:
Can you add `env = "LINERA_CLIENT_TOKIO_BLOCKING_THREADS"` ? This is useful for `linera net up`

Reply:
(also in the other places)